velox/exec/GroupingSet.cpp

```diff
@@ -300,6 +333,12 @@ std::vector<Accumulator> GroupingSet::accumulators() {
   if (sortedAggregations_ != nullptr) {
     accumulators.push_back(sortedAggregations_->accumulator());
   }
```

Comment:
NYC: at l330: shall we add explicit to  Accumulator(Aggregate* aggregate) to enforce an explicit conversion from Aggregator* to Accumulator to ease code reading? Thanks!

Reply:
I know what you are taking about. Will make that change in a follow-up PR.

Reply:
Submitted PR #5640